games-vue-client/cypress/integration/gameList.spec.js

```diff
@@ -0,0 +1,9 @@
+context('Game List', () => {
+    beforeEach(() => {
+        cy.login();
+    })
+
+    it('should have a list of games', () => {
+        cy.get('.game-type').should('have.length', 18);
```

Comment:
This will be changing fairly often, maybe just assert that it should be more than 10 ?